src/test/osd/TestRados.cc

```diff
@@ -157,6 +157,16 @@ class WeightedTestGenerator : public TestOpGenerator
 	if ((_oid) % 2) {
 	  oid << " " << string(300, 'o');
 	}
+
+        if (context.oid_in_use.count(oid.str())) {
+          /* previous write is not finished */
+          op = NULL;
+          m_op--;
+          cout << m_op << " wait for completion of write op! " << std::endl;
+          usleep(500);
```

Comment:
what is this sleep for?

Reply:
@liewegas This might prevent busy waiting. For example, if oid 1 is being used, a delay occur. So, we don't know when write op is finished.